website/docs/reference/network-view.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Network view
+---
+
+import Figure from '@site/src/components/Figure/Figure.tsx'
+
+:::info Availability
+
+The network view was released in Unleash 4.21. It is available to Pro and Enterprise users.
+
+You must configure the [data source](#data-source) to activate the feature.
+
+:::
+
+The Unleash admin UI contains a **network view** as part of the admin configuration pages. This network view was designed to give you an overview and understanding of incoming requests to your Unleash instance. It makes it possible to pinpoint sources of sudden request surges, and can therefore also help you identify issues with SDK setups[^1].
+
+The network view offers two different visualizations of the same data, known as the [network overview](#network-overview) and the [network traffic](#network-traffic) views.
+
+Because Unleash doesn't store this kind of data itself, the network view requires you to configure an [external data source](#data-source) for the network overview to work. The network view is only mad available if you tell Unleash where it can find the data (refer to the [data source section](#data-source)).
+
+The network view is intended to provide a simple and Unleash-centric overview that serves basic use cases. If you need detailed metrics and connection graphs, you may be better off using specialized network monitoring tools.
+
+## Applications
+
+Both the network overview and the network traffic diagrams show you **applications** that have made requests to the Unleash instance. An **application** is defined as anything that sends requests to [Unleash client API](/reference/api/unleash/client), such as [Unleash SDKs](./sdks/index.md), [Unleash Edge](/docs/generated/unleash-edge.md), or the [Unleash proxy](/docs/generated/unleash-proxy.md).
```

Comment:
```suggestion
Both the network overview and the network traffic diagrams show you **applications** that have made requests to the Unleash instance. An **application** is defined as anything that sends requests to [Unleash client API](/reference/api/unleash/client), such as [Unleash SDKs](./sdks/index.md), [Unleash Edge](/docs/generated/unleash-edge.md), or the [Unleash proxy](/docs/generated/unleash-proxy.md), even the admin UI.
```
Admin UI is one of those cases of unknown applications because it doesn't really matter much, but we wanted to have it anyway

Reply:
Got it! In that case, how about this? Because it's anything that sends something to `/api/admin` too, right?

```suggestion
Both the network overview and the network traffic diagrams show you **applications** that have made requests to the Unleash instance. An **application** is defined as anything that sends requests to the [Unleash client API](/reference/api/unleash/client) or the [Unleash admin API](/reference/api/unleash/admin). This includes as [Unleash SDKs](./sdks/index.md), [Unleash Edge](/docs/generated/unleash-edge.md), the [Unleash proxy](/docs/generated/unleash-proxy.md), and even the admin UI.
```

Reply:
Anything that sends requests to `/api/` whether it exist now or anything we create in the future. So `/api/frontend` too

Reply:
Got it! I think this clarifies that a little better.

```suggestion
Both the network overview and the network traffic diagrams show you **applications** that have made requests to the Unleash instance. An **application** is defined as anything that sends requests to the [Unleash client API](/reference/api/unleash/client), the [Unleash admin API](/reference/api/unleash/admin), the [Unleash front-end API](front-end-api.md), or any other API that Unleash exposes. This includes [Unleash SDKs](./sdks/index.md), [Unleash Edge](/docs/generated/unleash-edge.md), the [Unleash proxy](/docs/generated/unleash-proxy.md), and even the admin UI.
```